whist/server/api/game/action.py

```diff
@@ -11,6 +11,16 @@
 from whist.server.services.authentication import get_current_user
 from whist.server.services.game_db_service import GameDatabaseService
 
+import logging
+import sys
+
+logger = logging.getLogger(__name__)
+stream_handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter("%(levelname)s:%(message)s")
+stream_handler.setFormatter(formatter)
+logger.addHandler(stream_handler)
+logger.setLevel(logging.INFO)
```

Comment:
The logging setting shouldn't be part of the API. This is set during the start-up and therefore out of the scope of this PR.
Also on a side note, try to avoid writing duplicate code.